time/bits/types/struct_timespec.h

```diff
@@ -8,26 +8,17 @@
 
 /* POSIX.1b structure for a time value.  This is like a `struct timeval' but
    has nanoseconds instead of microseconds.  */
+  // Lind-Wasm: Original glibc code removed for compatibility
+  // to find original source code refer to (2.39.9000) at (time/bits/types/struct_timespec.h):(LINE 11-33)
+
 struct timespec
 {
-#ifdef __USE_TIME_BITS64
-  __time64_t tv_sec;		/* Seconds.  */
-#else
-  __time_t tv_sec;		/* Seconds.  */
-#endif
-#if __WORDSIZE == 64 \
-  || (defined __SYSCALL_WORDSIZE && __SYSCALL_WORDSIZE == 64) \
-  || (__TIMESIZE == 32 && !defined __USE_TIME_BITS64)
-  __syscall_slong_t tv_nsec;	/* Nanoseconds.  */
-#else
-# if __BYTE_ORDER == __BIG_ENDIAN
-  int: 32;           /* Padding.  */
+  time_t tv_sec;		/* Seconds.  */
+  int: 32;
+  // __int32_t __padding;           /* Padding.  */
```

Comment:
why are these commented out?

Reply:
I dont see any specific reason for this, I think int: 32 is equivelent with declare a __int32_t variable. I just reverted this part